README.md

```diff
@@ -110,10 +110,10 @@ The next step is to build an application you can combine with your bootloader to
 
 1. Connect the SD card to your computer.
 1. Copy the application binary (``<program name>_application.bin``, we defined it earlier as ``mbed-os-example-blinky_application.bin``) to the root of the SD card.
-1. Remove the SD card from your PC, and plug it into the mbed board.
+1. Remove the SD card from your PC. Ensure the CI test shield is attached to your device, and then plug the SD card into the CI test shield.
```

Comment:
The CI test shield isn't actually necessary here. Some boards have SD card slots built in, others can be added via various SD card shields/breakouts. I'd recommend leaving this line as is.